README.md

```diff
@@ -1,288 +1,133 @@
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+-[Live Version](https://world-kx2q.onrender.com/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
-
-### Prerequisites
-
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
-
-### Setup
-
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
+- Create a local directory that you want to clone the repository.
 
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
+- Open the command prompt in the created directory.
 
-### Install
+- On the terminal run this command git clone ``
 
-Install this project with:
+- Go to the repository folder using command prompt cd `World-info-React-Capstone`
 
-<!--
-Example command:
+- Install the dev dependencies for linters run npm install.
+- setup Webpack
 
-```sh
-  cd my-project
-  gem install
-```
---->
-
-### Usage
-
-To run the project, execute the following command:
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!--
-Example command:
+### Prerequisites
 
-```sh
-  rails server
-```
---->
+In order to run this project you need:
 
-### Run tests
+-Any broswer
 
-To run tests, run the following command:
+- knowledge of ES6 modules
+- familiarity with basic git commands
+- working knowledge of webpack
 
-<!--
-Example command:
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+### Setup
 
-### Deployment
+Clone this repository to your desired folder:
 
-You can deploy this project using:
+- Open the command prompt in the created directory.
 
-<!--
-Example:
+- On the terminal run this command git clone https://github.com/SabaAhmad404/World-info-React-Capstone
 
-```sh
+- Go to the repository folder using command prompt cd World-info-React-Capstone
 
-```
- -->
+- Install the dev dependencies for linters run npm install.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
+<!-- Author -->
 
-## 👥 Authors <a name="authors"></a>
+👤 **Authors**
+Outkarm Johnson
 
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [Outkarm](https://github.com/Outkarm)
+- GitHub: [Outkarm](https://github.com/Outkarm)
+- LinkedIn: [John Kpordje](https://www.linkedin.com/in/john-kpordje-866749241/)
```

Comment:
Mandatory

- your app should have unit tests for several components of the app and some pure functions in Redux code.
your app should have integration tests using React Testing Library
   * you should perform a mocking data fetching and Redux Store connection.